git-bin/git-bin.csproj

```diff
@@ -90,7 +94,9 @@
     <None Include="app.config" />
     <None Include="packages.config" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Analyzer Include="..\packages\AWSSDK.S3.3.1.8.0\analyzers\dotnet\cs\AWSSDK.S3.CodeAnalysis.dll" />
+  </ItemGroup>

```

Comment:
Some extra stuff came along for the ride when I installed the new AWS SDK using NuGet. I got rid of any large files but didn't mess with the small files (this one, the *.ps1 and *.xml files.)